### DISCUSSION TITLE
Consulta enunciado TPI 3

### DISCUSSION THREAD

**Discussion by Schmidt-mat** [dds-jv/foro#379] [Consultas generales] [answered]
Buenas noches, tengo una duda con respecto al enunciado. Al final dice "No olvidar hacer en papel el diagrama de clases", eso quedó del enunciado original pero en esta entrega no es necesario verdad? Pregunto porque talvez nos pide adjuntar una imagen o algo por el estilo, pero como se corrige automáticamente...

**Comment by gastonprieto** [comment, marked as answer]
@Schmidt-mat 

Si, eso quedo del enunciado original, no hace falta el diagrama
